src/it_depends/pip.py

```diff
@@ -231,6 +231,9 @@ def resolve(self, dependency: Dependency) -> Iterator[Package]:
         except ValueError as e:
             log.warning(str(e))
             return iter(())
+        except Exception:
```

Comment:
I'm a little concerned about a blanket exception handler; we should probably at least let keyboard interrupts propagate up:
```suggestion
        except KeyboardInterrupt:
            raise
        except Exception:
```